google-cloud-speech/samples/acceptance/speech_samples_test.rb

```diff
@@ -0,0 +1,163 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google/cloud/storage"
+require_relative "helper"
+require_relative "../speech_samples.rb"
+
+describe "Google Cloud Speech API samples" do
+  parallelize_me!
+  before do
+    @bucket_name = ENV["GCLOUD_TEST_STORAGE_BUCKET"]
```

Comment:
@dazuma Is there a Cloud Storage bucket that's used by google-cloud-ruby tests? If not, I may remove the upload step to rely on storage-samples bucket.